BlueChip/chain.json

```diff
@@ -0,0 +1,80 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "BlueChip",
+    "status": "live",
+    "website": "https://www.bluechip.link",
+    "network_type": "mainnet",
+    "pretty_name": "BlueChip",
```

Comment:
Although the chain_name property cannot have capital letters, the pretty_name is what apps will display and can keep the capital letters